tests/php/Forms/CurrencyField_DisabledTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace SilverStripe\Forms\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Forms\CurrencyField_Disabled;
+use SilverStripe\ORM\FieldType\DBCurrency;
+
+class CurrencyField_DisabledTest extends SapphireTest
```

Comment:
Ah there's actually a bunch of these 😴

Reply:
I think I've caught them all now, thanks!